.github/workflows/deploy-previews.yml

```
@@ -1065,6 +1071,74 @@ jobs:
+            --cpu=1000m \
+            --memory=1024Mi \
+            --update-secrets=/etc/env/.env=sta-env-superset:latest \
+            --command="/bin/sh","-c","cat /etc/env/.env >> /app/.env; /app/docker-entrypoint
+
+  superset-pr-comment:
+    name: superset-preview-link-comment
+    if: needs.check.outputs.superset == 'true'
+    needs: [superset]
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/github-script@v6
```

actionlint (1.7.4)

the runner of "actions/github-script@v6" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)